src/Utilities/ResourceHandle.h

```diff
@@ -27,7 +27,7 @@ class ResourceHandle : private std::optional<std::reference_wrapper<RS>>
 
   ResourceHandle() = default;
   ResourceHandle(RS& res) { Base::emplace(res); }
-  ResourceHandle& operator=(RS& res)
+  ResourceHandle& operator=(const RS& res)
```

Comment:
This is not a conventional assignment operator. Resources needs be modifiable and thus it should not carry const. I noticed this function not being used anywhere, we can probably just delete it.

Reply:
Removed